src/core/createEventManager.js

```diff
@@ -73,14 +73,15 @@ export default ({
      */
     sendEvent(event, options = {}) {
       event.setLastChanceCallback(onBeforeEventSendWithLoggedExceptions);
-      const { isViewStart = false } = options;
+      const { isViewStart = false, scopes } = options;
```

Comment:
I don't think `scopes` will ever be defined unless you change https://github.com/adobe/alloy/pull/335/files#diff-bd7d44476add132a5476024d3975c3f8R59-R61 to pass `scopes` into `eventManager.sendEvent`.